### ISSUE TITLE
Etcdwork tick reactor duration panel is not accurate enough to reflect the time cost

### ISSUE THREAD

**Issue by 3AceShowHand** [pingcap/tiflow#6900]
<img width="1473" alt="image" src="https://user-images.githubusercontent.com/7138436/186814134-de12cc25-859b-47c4-a70b-0ae84048edd6.png">

```
[2022/08/26 09:53:09.736 +08:00] [WARN] [processor.go:502] ["processor tick took too long"] [changefeed=blackhole-test] [capture=b3dd2fe5-20cf-439a-b2a5-4a81b49f1ef8] [duration=5.300958383s]
[2022/08/26 09:53:09.737 +08:00] [WARN] [etcd_worker.go:269] ["EtcdWorker reactor tick took too long"] [duration=5.302440407s] [role=processor]
```

we should use `p999` only to reflect the time cost, also distinguish `owner` and `processor`

**Comment by 3AceShowHand** [comment]
```
[2022/08/26 09:53:04.382 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=blackhole-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178805,\"region_epoch\":{\"conf_ver\":35,\"version\":953},\"checkpoint_ts\":435546694346866690,\"start_key\":\"dIAAAAAAAAH/219ygAAAAAD/XfDFAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/219ygAAAAAD/XjOkAAAAAAD6\",\"request_id\":2554915,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-7.jinling.tispace:20160]
[2022/08/26 09:53:04.382 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=blackhole-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":13448257,\"region_epoch\":{\"conf_ver\":35,\"version\":953},\"checkpoint_ts\":435546694346866690,\"start_key\":\"dIAAAAAAAAH/219ygAAAAAD/XjOkAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/219ygAAAAAD/XnZ5AAAAAAD6\",\"request_id\":2554916,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-7.jinling.tispace:20160]
[2022/08/26 09:53:04.383 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=blackhole-test] [regionID=15178805] [requestID=2554915] [span="[7480000000000001ffdb5f728000000000ff5df0c50000000000fa, 7480000000000001ffdb5f728000000000ff5e33a40000000000fa)"] [checkpoint=435546694346866690] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: region_not_found:<region_id:15178805 > "] [sri={}]
[2022/08/26 09:53:04.383 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-blackhole-test] [lockID=115] [regionID=15178805] [checkpointTs=435546694346866690] [startKey=7480000000000001ffdb5f728000000000ff5df0c50000000000fa] [endKey=7480000000000001ffdb5f728000000000ff5e33a40000000000fa]
[2022/08/26 09:53:04.384 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=115] [regionID=15178805] [version=953] [checkpointTs=435546694346866690] [startKey=7480000000000001ffdb5f728000000000ff5df0c50000000000fa] [endKey=7480000000000001ffdb5f728000000000ff5e33a40000000000fa]
[2022/08/26 09:53:04.384 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=blackhole-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178805,\"region_epoch\":{\"conf_ver\":35,\"version\":953},\"checkpoint_ts\":435546694346866690,\"start_key\":\"dIAAAAAAAAH/219ygAAAAAD/XfDFAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/219ygAAAAAD/XjOkAAAAAAD6\",\"request_id\":2554917,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-7.jinling.tispace:20160]
[2022/08/26 09:53:06.748 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=mysql-test] [regionID=12416534] [requestID=2554414] [span="[7480000000000001ffdd5f728000000000ff5677370000000000fa, 7480000000000001ffdd5f728000000000ff56fcf50000000000fa)"] [checkpoint=435546694451724315] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: epoch_not_match:<current_regions:<id:15178821 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377Vw7\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377V\\272\\026\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:17 version:970 > peers:<id:15178822 store_id:1 > peers:<id:15178823 store_id:4 > peers:<id:15178824 store_id:5 > > current_regions:<id:12416534 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377V\\272\\026\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377V\\374\\365\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:17 version:970 > peers:<id:12416535 store_id:1 > peers:<id:12416536 store_id:4 > peers:<id:12416537 store_id:5 > > > "] [sri={}]
[2022/08/26 09:53:06.845 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-mysql-test] [lockID=160] [regionID=12416534] [checkpointTs=435546694451724315] [startKey=7480000000000001ffdd5f728000000000ff5677370000000000fa] [endKey=7480000000000001ffdd5f728000000000ff56fcf50000000000fa]
[2022/08/26 09:53:07.337 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=160] [regionID=15178821] [version=970] [checkpointTs=435546694451724315] [startKey=7480000000000001ffdd5f728000000000ff5677370000000000fa] [endKey=7480000000000001ffdd5f728000000000ff56ba160000000000fa]
[2022/08/26 09:53:07.337 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=160] [regionID=12416534] [version=970] [checkpointTs=435546694451724315] [startKey=7480000000000001ffdd5f728000000000ff56ba160000000000fa] [endKey=7480000000000001ffdd5f728000000000ff56fcf50000000000fa]
[2022/08/26 09:53:07.338 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178821,\"region_epoch\":{\"conf_ver\":17,\"version\":970},\"checkpoint_ts\":435546694451724315,\"start_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/Vnc3AAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/VroWAAAAAAD6\",\"request_id\":2554918,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:07.338 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":12416534,\"region_epoch\":{\"conf_ver\":17,\"version\":970},\"checkpoint_ts\":435546694451724315,\"start_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/VroWAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/Vvz1AAAAAAD6\",\"request_id\":2554919,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:08.443 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=mysql-test] [regionID=15046693] [requestID=2554401] [span="[7480000000000001ffdd5f728000000000ff6c25880000000000fa, 7480000000000001ffdd5f728000000000ff6cee260000000000fa)"] [checkpoint=435546694399295595] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: epoch_not_match:<current_regions:<id:15178813 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377l%\\210\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377lhg\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:17 version:1083 > peers:<id:15178814 store_id:1 > peers:<id:15178815 store_id:4 > peers:<id:15178816 store_id:5 > > current_regions:<id:15178817 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377lhg\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377l\\253G\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:17 version:1083 > peers:<id:15178818 store_id:1 > peers:<id:15178819 store_id:4 > peers:<id:15178820 store_id:5 > > current_regions:<id:15046693 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377l\\253G\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\335_r\\200\\000\\000\\000\\000\\377l\\356&\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:17 version:1083 > peers:<id:15046694 store_id:1 > peers:<id:15046695 store_id:4 > peers:<id:15046696 store_id:5 > > > "] [sri={}]
[2022/08/26 09:53:08.444 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-mysql-test] [lockID=160] [regionID=15046693] [checkpointTs=435546694399295595] [startKey=7480000000000001ffdd5f728000000000ff6c25880000000000fa] [endKey=7480000000000001ffdd5f728000000000ff6cee260000000000fa]
[2022/08/26 09:53:08.643 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=160] [regionID=15178813] [version=1083] [checkpointTs=435546694399295595] [startKey=7480000000000001ffdd5f728000000000ff6c25880000000000fa] [endKey=7480000000000001ffdd5f728000000000ff6c68670000000000fa]
[2022/08/26 09:53:08.643 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=160] [regionID=15178817] [version=1083] [checkpointTs=435546694399295595] [startKey=7480000000000001ffdd5f728000000000ff6c68670000000000fa] [endKey=7480000000000001ffdd5f728000000000ff6cab470000000000fa]
[2022/08/26 09:53:08.643 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=160] [regionID=15046693] [version=1083] [checkpointTs=435546694399295595] [startKey=7480000000000001ffdd5f728000000000ff6cab470000000000fa] [endKey=7480000000000001ffdd5f728000000000ff6cee260000000000fa]
[2022/08/26 09:53:08.643 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178813,\"region_epoch\":{\"conf_ver\":17,\"version\":1083},\"checkpoint_ts\":435546694399295595,\"start_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/bCWIAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/bGhnAAAAAAD6\",\"request_id\":2554920,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-6.jinling.tispace:20160]
[2022/08/26 09:53:08.644 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178817,\"region_epoch\":{\"conf_ver\":17,\"version\":1083},\"checkpoint_ts\":435546694399295595,\"start_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/bGhnAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/bKtHAAAAAAD6\",\"request_id\":2554921,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-6.jinling.tispace:20160]
[2022/08/26 09:53:08.645 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15046693,\"region_epoch\":{\"conf_ver\":17,\"version\":1083},\"checkpoint_ts\":435546694399295595,\"start_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/bKtHAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/3V9ygAAAAAD/bO4mAAAAAAD6\",\"request_id\":2554922,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-6.jinling.tispace:20160]
[2022/08/26 09:53:08.740 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=mysql-test] [regionID=14887049] [requestID=2554177] [span="[7480000000000001ffc95f728000000000ff3ed8900000000000fa, 7480000000000001ffc95f728000000000ff3fa12f0000000000fa)"] [checkpoint=435546695421657217] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: epoch_not_match:<current_regions:<id:15178981 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\311_r\\200\\000\\000\\000\\000\\377>\\330\\220\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\311_r\\200\\000\\000\\000\\000\\377?\\033o\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:5 version:889 > peers:<id:15178982 store_id:1 > peers:<id:15178983 store_id:4 > peers:<id:15178984 store_id:5 > > current_regions:<id:15178985 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\311_r\\200\\000\\000\\000\\000\\377?\\033o\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\311_r\\200\\000\\000\\000\\000\\377?^O\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:5 version:889 > peers:<id:15178986 store_id:1 > peers:<id:15178987 store_id:4 > peers:<id:15178988 store_id:5 > > current_regions:<id:14887049 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\311_r\\200\\000\\000\\000\\000\\377?^O\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\311_r\\200\\000\\000\\000\\000\\377?\\241/\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:5 version:889 > peers:<id:14887050 store_id:1 > peers:<id:14887051 store_id:4 > peers:<id:14887052 store_id:5 > > > "] [sri={}]
[2022/08/26 09:53:08.740 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-mysql-test] [lockID=153] [regionID=14887049] [checkpointTs=435546695421657217] [startKey=7480000000000001ffc95f728000000000ff3ed8900000000000fa] [endKey=7480000000000001ffc95f728000000000ff3fa12f0000000000fa]
[2022/08/26 09:53:08.744 +08:00] [WARN] [pd.go:152] ["get timestamp too slow"] ["cost time"=195.681049ms]
[2022/08/26 09:53:08.750 +08:00] [WARN] [system.go:578] ["actor poll received messages too slow"] [duration=4.315714857s] [id=131] [name=sorter-writer]
[2022/08/26 09:53:08.837 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=153] [regionID=15178981] [version=889] [checkpointTs=435546695421657217] [startKey=7480000000000001ffc95f728000000000ff3ed8900000000000fa] [endKey=7480000000000001ffc95f728000000000ff3f1b6f0000000000fa]
[2022/08/26 09:53:08.837 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=153] [regionID=15178985] [version=889] [checkpointTs=435546695421657217] [startKey=7480000000000001ffc95f728000000000ff3f1b6f0000000000fa] [endKey=7480000000000001ffc95f728000000000ff3f5e4f0000000000fa]
[2022/08/26 09:53:08.837 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=153] [regionID=14887049] [version=889] [checkpointTs=435546695421657217] [startKey=7480000000000001ffc95f728000000000ff3f5e4f0000000000fa] [endKey=7480000000000001ffc95f728000000000ff3fa12f0000000000fa]
[2022/08/26 09:53:08.838 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178981,\"region_epoch\":{\"conf_ver\":5,\"version\":889},\"checkpoint_ts\":435546695421657217,\"start_key\":\"dIAAAAAAAAH/yV9ygAAAAAD/PtiQAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/yV9ygAAAAAD/PxtvAAAAAAD6\",\"request_id\":2554923,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:08.838 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178985,\"region_epoch\":{\"conf_ver\":5,\"version\":889},\"checkpoint_ts\":435546695421657217,\"start_key\":\"dIAAAAAAAAH/yV9ygAAAAAD/PxtvAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/yV9ygAAAAAD/P15PAAAAAAD6\",\"request_id\":2554924,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:08.838 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":14887049,\"region_epoch\":{\"conf_ver\":5,\"version\":889},\"checkpoint_ts\":435546695421657217,\"start_key\":\"dIAAAAAAAAH/yV9ygAAAAAD/P15PAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/yV9ygAAAAAD/P6EvAAAAAAD6\",\"request_id\":2554925,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:08.844 +08:00] [WARN] [system.go:578] ["actor poll received messages too slow"] [duration=4.409913882s] [id=112] [name=sorter-writer]
[2022/08/26 09:53:09.046 +08:00] [WARN] [system.go:578] ["actor poll received messages too slow"] [duration=4.610790573s] [id=134] [name=sorter-writer]
[2022/08/26 09:53:09.135 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=mysql-test] [regionID=15039237] [requestID=2432536] [span="[7480000000000002ff0b5f728000000000ff16c9010000000000fa, 7480000000000002ff0b5f728000000000ff16cf6b0000000000fa)"] [checkpoint=435546695421657104] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: epoch_not_match:<> "] [sri={}]
[2022/08/26 09:53:09.135 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-mysql-test] [lockID=208] [regionID=15039237] [checkpointTs=435546695421657104] [startKey=7480000000000002ff0b5f728000000000ff16c9010000000000fa] [endKey=7480000000000002ff0b5f728000000000ff16cf6b0000000000fa]
[2022/08/26 09:53:09.136 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=mysql-test] [regionID=15039241] [requestID=2432537] [span="[7480000000000002ff0b5f728000000000ff16cf6b0000000000fa, 7480000000000002ff0b5f728000000000ff16d5c20000000000fa)"] [checkpoint=435546695421657104] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: epoch_not_match:<> "] [sri={}]
[2022/08/26 09:53:09.136 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-mysql-test] [lockID=208] [regionID=15039241] [checkpointTs=435546695421657104] [startKey=7480000000000002ff0b5f728000000000ff16cf6b0000000000fa] [endKey=7480000000000002ff0b5f728000000000ff16d5c20000000000fa]
[2022/08/26 09:53:09.235 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=208] [regionID=15039241] [version=852] [checkpointTs=435546695421657104] [startKey=7480000000000002ff0b5f728000000000ff16c9010000000000fa] [endKey=7480000000000002ff0b5f728000000000ff16d5c20000000000fa]
[2022/08/26 09:53:09.236 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=mysql-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15039241,\"region_epoch\":{\"conf_ver\":17,\"version\":852},\"checkpoint_ts\":435546695421657104,\"start_key\":\"dIAAAAAAAAL/C19ygAAAAAD/FskBAAAAAAD6\",\"end_key\":\"dIAAAAAAAAL/C19ygAAAAAD/FtXCAAAAAAD6\",\"request_id\":2554926,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-6.jinling.tispace:20160]
[2022/08/26 09:53:09.236 +08:00] [INFO] [region_range_lock.go:255] ["tryLockRange stale"] [changefeed=default-mysql-test] [lockID=208] [regionID=15039241] [startKey=7480000000000002ff0b5f728000000000ff16c9010000000000fa] [endKey=7480000000000002ff0b5f728000000000ff16d5c20000000000fa] [allOverlapping="[\"regionID: 15039241, ver: 852, start: 7480000000000002ff0b5f728000000000ff16c9010000000000fa, end: 7480000000000002ff0b5f728000000000ff16d5c20000000000fa\"]"]
[2022/08/26 09:53:09.236 +08:00] [INFO] [client.go:626] ["request expired"] [namespace=default] [changefeed=mysql-test] [regionID=15039241] [span="[7480000000000002ff0b5f728000000000ff16c9010000000000fa, 7480000000000002ff0b5f728000000000ff16d5c20000000000fa)"] [retrySpans="[]"] [sri={}]
[2022/08/26 09:53:09.339 +08:00] [INFO] [region_worker.go:253] ["single region event feed disconnected"] [namespace=default] [changefeed=blackhole-test] [regionID=11886490] [requestID=2554439] [span="[7480000000000001ffe35f728000000000ff63c01a0000000000fa, 7480000000000001ffe35f728000000000ff6445d60000000000fa)"] [checkpoint=435546694671400976] [error="[CDC:ErrEventFeedEventError]eventfeed returns event error: epoch_not_match:<current_regions:<id:15178845 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\343_r\\200\\000\\000\\000\\000\\377c\\300\\032\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\343_r\\200\\000\\000\\000\\000\\377d\\002\\371\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:23 version:1010 > peers:<id:15178846 store_id:5 > peers:<id:15178847 store_id:1 > peers:<id:15178848 store_id:4 > > current_regions:<id:11886490 start_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\343_r\\200\\000\\000\\000\\000\\377d\\002\\371\\000\\000\\000\\000\\000\\372\" end_key:\"t\\200\\000\\000\\000\\000\\000\\001\\377\\343_r\\200\\000\\000\\000\\000\\377dE\\326\\000\\000\\000\\000\\000\\372\" region_epoch:<conf_ver:23 version:1010 > peers:<id:11886491 store_id:5 > peers:<id:11886492 store_id:1 > peers:<id:11886493 store_id:4 > > > "] [sri={}]
[2022/08/26 09:53:09.339 +08:00] [INFO] [region_range_lock.go:389] ["unlocked range"] [changefeed=default-blackhole-test] [lockID=136] [regionID=11886490] [checkpointTs=435546694671400976] [startKey=7480000000000001ffe35f728000000000ff63c01a0000000000fa] [endKey=7480000000000001ffe35f728000000000ff6445d60000000000fa]
[2022/08/26 09:53:09.436 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=136] [regionID=15178845] [version=1010] [checkpointTs=435546694671400976] [startKey=7480000000000001ffe35f728000000000ff63c01a0000000000fa] [endKey=7480000000000001ffe35f728000000000ff6402f90000000000fa]
[2022/08/26 09:53:09.436 +08:00] [INFO] [region_range_lock.go:224] ["range locked"] [lockID=136] [regionID=11886490] [version=1010] [checkpointTs=435546694671400976] [startKey=7480000000000001ffe35f728000000000ff6402f90000000000fa] [endKey=7480000000000001ffe35f728000000000ff6445d60000000000fa]
[2022/08/26 09:53:09.436 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=blackhole-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":15178845,\"region_epoch\":{\"conf_ver\":23,\"version\":1010},\"checkpoint_ts\":435546694671400976,\"start_key\":\"dIAAAAAAAAH/419ygAAAAAD/Y8AaAAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/419ygAAAAAD/ZAL5AAAAAAD6\",\"request_id\":2554927,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:09.436 +08:00] [INFO] [client.go:804] ["start new request"] [namespace=default] [changefeed=blackhole-test] [request="{\"header\":{\"cluster_id\":7133877136889696851,\"ticdc_version\":\"6.3.0-master\"},\"region_id\":11886490,\"region_epoch\":{\"conf_ver\":23,\"version\":1010},\"checkpoint_ts\":435546694671400976,\"start_key\":\"dIAAAAAAAAH/419ygAAAAAD/ZAL5AAAAAAD6\",\"end_key\":\"dIAAAAAAAAH/419ygAAAAAD/ZEXWAAAAAAD6\",\"request_id\":2554928,\"extra_op\":1,\"Request\":null}"] [addr=lingjin-5.jinling.tispace:20160]
[2022/08/26 09:53:09.736 +08:00] [WARN] [processor.go:502] ["processor tick took too long"] [changefeed=blackhole-test] [capture=b3dd2fe5-20cf-439a-b2a5-4a81b49f1ef8] [duration=5.300958383s]
[2022/08/26 09:53:09.737 +08:00] [WARN] [etcd_worker.go:269] ["EtcdWorker reactor tick took too long"] [duration=5.302440407s] [role=processor]
[2022/08/26 09:53:09.740 +08:00] [INFO] [statistics.go:198] ["sink replication status"] [sinkType=DB] [namespace=default] [changefeed=mysql-test] [capture=lingjin-4.jinling.tispace:8300] [count=0] [qps=0] [ddl=0]
[2022/08/26 09:53:09.740 +08:00] [INFO] [statistics.go:198] ["sink replication status"] [sinkType=DB] [namespace=default] [changefeed=mysql-test] [capture=lingjin-4.jinling.tispace:8300] [count=0] [qps=0] [ddl=0]
[2022/08/26 09:53:10.035 +08:00] [WARN] [pd.go:152] ["get timestamp too slow"] ["cost time"=100.774802ms]
```

this log should be helpful to locate the reason why a tick cost around 5s

**Comment by 3AceShowHand** [comment]
closed by #6929
